pkgs/development/interpreters/python/cpython/default.nix

```
@@ -240,6 +240,7 @@ in with passthru; stdenv.mkDerivation {
   ] ++ optionals (pythonAtLeast "3.9" && stdenv.isDarwin) [
     # Stop checking for TCL/TK in global macOS locations
     ./3.9/darwin-tcl-tk.patch
+  ] ++ optionals (isPy39 && stdenv.isDarwin) [
```

```suggestion
  ] ++ optionals (isPy39 && pythonOlder "3.9.7" && stdenv.isDarwin) [
```

It's possible we might backport more 3.9 releases in the future and this patch has made it into 3.9.7.

Other than that LGTM.

---

On second thought, this LGTM. The build will fail when we backport and we will remove the patch then.